packages/front-end/components/Experiment/EditMetricsForm.tsx

```diff
@@ -1,65 +1,141 @@
+        header="Edit Metrics"
+        size="lg"
+        open={true}
+        close={cancel}
+        submit={form.handleSubmit(async (value) => {
+          await apiCall(`/experiment/${experiment.id}`, {
+            method: "POST",
+            body: JSON.stringify(value),
+          });
+          mutate();
+        })}
+        cta="Save"
+      >
+        <div className="form-group">
+          <label className="font-weight-bold mb-1">Goal Metrics</label>
+          <div className="mb-1 font-italic">
+            Metrics you are trying to improve with this experiment.
+          </div>
+          <MetricsSelector
+            selected={form.watch("metrics")}
+            onChange={(metrics) => form.setValue("metrics", metrics)}
+            datasource={experiment.datasource}
+            autoFocus={true}
+          />
         </div>
-        <MetricsSelector
-          selected={form.watch("guardrails")}
-          onChange={(metrics) => form.setValue("guardrails", metrics)}
-          datasource={experiment.datasource}
-        />
-      </div>
-      <div style={{ height: 100 }} />
-    </Modal>
+
+        <div className="form-group">
+          <label className="font-weight-bold mb-1">Guardrail Metrics</label>
+          <div className="mb-1 font-italic">
+            Metrics you want to monitor, but are NOT specifically trying to
+            improve.
+          </div>
+          <MetricsSelector
+            selected={form.watch("guardrails")}
+            onChange={(metrics) => form.setValue("guardrails", metrics)}
+            datasource={experiment.datasource}
+          />
+        </div>
+
+        <div className="form-group">
+          <label className="font-weight-bold mb-1">Activation Metric</label>
+          <div className="mb-1 font-italic">
+            Users must convert on this metric before being included.
+          </div>
+          <SelectField
+            options={filteredMetrics.map((m) => {
+              return {
+                label: m.name,
+                value: m.id,
+              };
+            })}
+            initialOption="None"
+            value={form.watch("activationMetric")}
+            onChange={(metric) => form.setValue("activationMetric", metric)}
+          />
+        </div>
+
+        <div className="form-group mb-4">
+          <label className="font-weight-bold mb-1">
+            Metric Overrides (optional)
+          </label>
+          {!hasOverrideMetricsFeature && (
+            <UpgradeMessage
+              showUpgradeModal={() => setUpgradeModal(true)}
+              commercialFeature="override-metrics"
+              upgradeMessage="override metrics"
+            />
+          )}
+          <div className="mb-1 font-italic">
+            Override metric conversion windows within this experiment.
+          </div>
+          <MetricsOverridesSelector
+            experiment={experiment}
+            form={form}
+            disabled={!hasOverrideMetricsFeature}
+          />
```

Comment:
I like the concept of continuing to show the disabled UI, but I don't think it quite works here.  It doesn't actually look disabled (the dropdown still looks clickable, the label text isn't grayed out at all, etc.)

![image](https://user-images.githubusercontent.com/1087514/204630370-71e2474d-4f2a-462b-a2ff-bffd8b5aad25.png)

Reply:
Adjusted the styling